examples/from-dockerfile/2.7/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubi8/ruby-27
+
+USER 0
+ADD app-src ./
+
+RUN export PATH=${PATH}:/opt/rh/rh-ruby27/root/usr/bin/ && \
```

Comment:
This should not be ideally needed, but it is probably necessary until something like https://github.com/sclorg/s2i-nodejs-container/pull/230/files is adopted. So, I think we should first adopt this new concept in the parent Dockerfile (the ruby image itself), so layered Dockerfiles can use `ruby`/`bundler` without any special hacks.